src/main/kotlin/com/sourcegraph/cody/edit/FixupService.kt

```diff
@@ -81,6 +118,20 @@ class FixupService(val project: Project) : Disposable {
     activeSession = null
   }
 
+  private fun handlePolicyChange(context: IgnoreMessageBus.Context) {
```

Comment:
What behavior does this produce?

It doesn't seem right to have FixupService producing a notification for "any file that someone `policyForUri`-ed in the past hour(s)" (the last time we would have fetched the policy and started caching results for individual files.)